index.html

```diff
@@ -0,0 +1,179 @@
+							<figcaption>
+								My  photo.
+							</figcaption>
+						</figure>
+					</div>
+					<div>
+						<h3> <a id="PROFILE">PROFILE</a>	</h3>
+						<article>
+							<p> I am an open-minded individual with a  track record in designing websites and creating databases.
+								 I have  technical skills as well as excellent interpersonal skills. I am eager to be challenged 
+								 in order to grow and improve my communication and professional IT skills in the IT phere.
+								 I really like to learn and learn something new. I'm really good at combining University studies
+								  with learning programming languages like JS now.
+							</p>
+						</article>
+			
+					</div>
+			</section>
+			<section>
+					<div>
+						<h3><a id="EDUCATION">EDUCATION</a> </h3>
+						<article>
+								<ul>
+									<li>2016     	    English courses IH.</li>
+									<li>2016		    Java programming courses at BSU</li>
+									<li>2015		    C++ programming courses at BSU</li>
+									<li>2015-Present	Belarusian State University faculty of mechanics and mathematics</li>
+									<li>2004-2015		Shool №4 Kobrin, Belarus </li>		
+								</ul>	
+
+						</article>
+			
+					</div>
+			</section>
+			<section>
+					<div>
+						<h3><a id="LEADERSHIP ACTIVITIES AND AWARDS">LEADERSHIP ACTIVITIES AND AWARDS</a> </h3>
+						<article>	
+							<ul>
+								
+								<li>2016-2018       Deputy head of the student organization</li>
+								<li>2015-2018       I  participated in competitions on judo</li>
+								<li>2014		    Participant of the Olympiad in physics and mathematics</li>
+								<li>2014		    First-degree diploma at the regional competition on radionics.</li>
+								<li>2013		    Participant of the Olympiad in physics and mathematics</li>
+								<li>2013		    The third degree diploma at the regional competition on radionics</li>	
+										
+								</ul>	
+						
+						</article>
+			
+					</div>
+			</section>
+			<section>
+					<div>
+						<h3><a id="SKILLS SUMMARY">SKILLS SUMMARY</a> </h3>
+						<article>
+								<table border="2" cellpadding="10" cellspacing="10">
+										<tr>
+											<th>Web Design</th>
+											<th>HTML, CSS, JavaScript</th>
+										</tr>
+										<tr>
+											<th>Microsoft Office</th>
+											<th >Word, Excel, PowerPoint, Outlook</th>
+
+										</tr>
+										<tr>
+											<th>Programming Languages</th>
+											<th>C++, Java</th>
+										</tr>
+										<tr>
+												<th>Database</th>
+												<th>SQL</th>
+										</tr>
+										<tr>
+												<th>Engineering 3d modeling (computer mechanics)</th>
+												<th>Ansys, Solid Works, Femap</th>
+										</tr>
+									</table>
+							
+						</article>
+			
+					</div>
+			</section>
+			<section>
+					<div>
+						<h3> <a id="HOBBIES AND INTERESTS">HOBBIES AND INTERESTS</a> </h3>
+						<article>
+							<p>		•	Boxing,
+									•	Swimming, 
```

Comment:
If you need markers in the form of points, as a design element, especially in the enumerated elements, it is better to use special tags for this, such as (ul and li)
<ul> 
   <li> ... </ li>
   <li> ... </ li>
   <li > ... </ li>
</ ul>